Userland/Applications/Terminal/main.cpp

```diff
@@ -417,7 +425,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(Core::System::unveil(nullptr, nullptr));
 
     auto modified_state_check_timer = Core::Timer::create_repeating(500, [&] {
```

Comment:
if `should_confirm_close` is false, we'll end up with a useless timer repeating forever doing nothing, pls avoid that.